public/js/chat.js

```diff
@@ -1,18 +1,18 @@
-const hamburger = document.getElementsByClassName('huhamburger')[0];
-const aside = document.querySelectorAll('aside .hide_aside')[0];
-const side = document.getElementsByTagName('aside')[0];
-const alter_header = document.getElementsByClassName('alter_header')[0];
-const header = document.getElementsByTagName('header')[0];
-const messages = document.querySelectorAll('.main_section .last_messages ul')[0];
-const messenger_space = document.querySelector(' main .main_section .messenger');
-const last_messages = document.querySelector('main .main_section .last_messages');
-const back_btn = document.querySelector('.alter_header i');
+const hamburger = document.getElementById('hamburger');
+const sliderbar = document.getElementById('slider_bar');
+const aside = document.getElementById('aside');
+const alter_header = document.getElementById('alter_header');
+const header = document.getElementById('header');
+const messages = document.getElementById('messages');
+const messenger_space = document.getElementById('messenger_space');
+const last_messages = document.getElementById('last_messages');
+const back_btn = document.getElementById('back_btn');
 
 if (hamburger) {
```

Comment:
these lines (11-17) are the same code in the doctors.js file so you could make a single file for the two instead of repeating the same code twice